pkg/systemd/overview-cards/loginMessage.jsx

```diff
@@ -52,14 +53,20 @@ export class LoginMessage extends React.PureComponent {
     }
 
     render() {
-        const { time, host, line } = this.props;
+        const { time, host, line, text, username } = this.props;
         const datetime = this.getFormattedDateTime(time * 1000);
-        return this.generate_line(host, line, datetime);
+        return (
+            <>
+                <Button variant="link" isInline component="span" onClick={() => cockpit.jump(`/users#/${username}`)}>{username}</Button> {text} {this.generate_line(host, line, datetime)}
```

Comment:
Why did you add component='span' here? feels weird to have a span act like a button.

Reply:
Could you break the line? It's very long! I have to scroll right to see it on my laptop screen.

Reply:
> Why did you add component='span' here? feels weird to have a span act like a button.

I was going off the patternfly inline example.

Reply:
Which one? They have tons of examples just to show the possible use cases.